### PR TITLE
ReadMe Added Carbon.HttpClient.Auth

### PR DESCRIPTION
**Pull request by keremaytac**

### CODE REVIEW

**Review by osman-koc — APPROVED** [review]

File: Carbon.HttpClient.Auth/README.Md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+app.CreateAuthentication("RAM");
+```
+
+And use the configuration as given below.
+AuthenticationType 4 is type of *client_credentials* in *oAuth2*. AuthenticationType 1, 2 or 3 is legacy and unsupported.
+
+```json
+"Authorizations":
+{
+    "RAM":
```
Comment (616e696c) [inline Carbon.HttpClient.Auth/README.Md:41]: Can we add a comment here; people can add their client as RAM every time if we gave an example like that.

**Review by yaschete — APPROVED** [review]

File: Carbon.HttpClient.Auth/README.Md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+//IServiceCollection services
+services.AddGrpcHttpClientAuth();
+```
+
+**2. Create an authentication with the name just as it is writing in the configuration file**
+
+This creates the token at startup by communicating with identity server and stores it until you ask for.
+```csharp
+//IApplicationBuilder
+app.CreateAuthentication("RAM");
```
Comment (616e696c) [inline Carbon.HttpClient.Auth/README.Md:32]: How can we tell that this is an exemplary name?

**Review by 616e696c — CHANGES_REQUESTED** [review]

File: Carbon.HttpClient.Auth/README.Md (hunk 1 above)
Comment (keremaytac) [reply Carbon.HttpClient.Auth/README.Md:41]: Will be introduced in the next PR

File: Carbon.HttpClient.Auth/README.Md (hunk 2 above)
Comment (keremaytac) [reply Carbon.HttpClient.Auth/README.Md:32]: Will be introduced in the next PR

**Pull request merged**